settings.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
@snova-kwasia, @snova-jorgep 
We probably should not commit the VSCode `settings.json`.
Everyone should be free to configure it as they wish.

Reply:
We can move this to Confluence.

Reply:
I agree with this, is better to have among the explanations you have on the confluence page

Reply:
Yep agreed that confluence is the best place for this.